mathics/builtin/drawing/drawing_options.py

```diff
@@ -78,6 +78,32 @@ class Axis(Builtin):
     summary_text = "graph option value to fill plot from curve to the axis"
 
 
+class Background(Builtin):
+    """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Background.html</url>
+
+    <dl>
+      <dt>'Background'
+      <dd>is an option that specifies the color of the background.
+    </dl>
+
+    The specification must be a 'Color' or 'Automatic':
+
+    >> Graphics3D[{Arrow[{{0,0,0},{1,0,1},{0,-1,0},{1,1,1}}]}, Background -> Red]
+     = -Graphics3D-
+
+    Other values for this option are neglected. For example, this prevent to
```

Comment:
> For example, this prevent[s] to comb[ing]? a color with an opacity. 

I don't know what this means. What is an example of something that is "prevented"?

Reply:
The point is that (in WMA) `Background->{Red,Opacity[.2]}` is not a valid background specification. To produce that effect,  `Background->{RGBColor[1,0,0,.25]}` must be used instead. Does it make sense to put this information in the documentation?

Reply:
The reworded text makes things clearer now. Thanks.

Reply:
@rocky, if you are up to it and you want to improve or extend the documentation, feel free to push your changes over this.

Reply:
Ok. I will probably make some small changes when I go over all of the documentation in the phase where I try to  build documentation before a release. And probably add a couple svg and asy tests too. 

So I guess we can merge and iterate if this is all you have time for, for now. 

Currently, I'm working on a bug in getting documentation in Django that causes malfunction after Mathics3 modules load.

Reply:
Ok. Then I am going to add a pytest for the SVG formatter, and then I am going to merge this.

Reply:
And one for `asy` as well please?

Reply:
This is done already.

Reply:
> This is done already.

I am not seeing any changes in https://github.com/Mathics3/mathics-core/pull/969/files any changes to `test/format/test_asy.py`

Reply:
You are right. I put it together with other tests coming from private doctests. I will move them to this other module.